snapshots/ggdc/2022-12-23/maddison_database_2010.xlsx.dvc

```diff
@@ -0,0 +1,21 @@
+meta:
+  namespace: ggdc
+  short_name: maddison_database_2010
+  name: Maddison Database (GGDC, 2009)
+  source_name: Maddison Database (Maddison, 2009)
+  publication_year: 2009
+  publication_date: 2009
```

Comment:
I preferred to fill it because when empty it failed for meadow. I remember the folders were something like `ggdc/1988/maddison...`. I will try to replicate it.